integrations/triton/mcp-handler.py

```
@@ -153,127 +166,172 @@
         try:
             key = self._get_key(context_id)
-            self.redis_client.delete(key)
+            await self.redis_client.delete(key)
             return True
         except Exception as e:
             logger.error(f"Error deleting context {context_id}: {e}")
             return False
 
     async def list_contexts(self, model_name: Optional[str] = None) -> List[str]:
         """List all context IDs, optionally filtered by model."""
+        if not self.redis_client: await self.connect_redis()
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)